### PR TITLE
feat: settling aave pools

### PR DESCRIPTION
**Pull request by ioanaroceanu2000**
# Title

Enable settling of Aave pools

https://www.notion.so/voltz/Enable-settlement-of-paused-markets-f8a12b8cbae14064870234f7157be1f0

### CODE REVIEW

**Review by avniculae — APPROVED** [review]

File: src/app/features/settle-flow/selectors.ts [hunk 1]
```diff
@@ -162,3 +164,14 @@ export const selectIsGLP28June = (state: RootState): boolean => {
 
   return pool.flags.isGLP28Jun2023;
 };
+
+export const selectIsAaveAugust = (state: RootState): boolean => {
```
Comment (ssbarbee) [inline src/app/features/settle-flow/selectors.ts:168]:
```suggestion
export const selectIsSettlementAllowedWhenPaused; = (state: RootState): boolean => {
```

File: src/app/features/settle-flow/selectors.ts [hunk 2]
```diff
@@ -162,3 +164,14 @@ export const selectIsGLP28June = (state: RootState): boolean => {
...
+  const pool = state.settleFlow.position?.pool;
+
+  if (!pool) {
+    return false;
+  }
+
+  // todo: rename this flag
```
Comment (ssbarbee) [inline src/app/features/settle-flow/selectors.ts:175]: remove todo or do it now :)

File: src/ui/components/SettleFlow/SettleConfirmationStep/index.tsx [hunk 3]
```diff
@@ -64,21 +64,21 @@ export const SettleConfirmationStep: React.FunctionComponent = () => {
       </FeeDetailsBox>
       <Button
         bottomLeftText={
-          isAMMPaused
+          !isSettlementAllowed
             ? 'Pool paused until further notice. Check Discord for updates.'
```
Comment (ssbarbee) [inline src/ui/components/SettleFlow/SettleConfirmationStep/index.tsx:68]:
```suggestion
            ? 'Pool settlement paused until further notice. Check Discord for updates.'
```

**Review by ssbarbee — CHANGES_REQUESTED** [review]
reduce the amount of changes in yarn.lock... this can break entire application. 

seems as stuff was coming from registry.npmjs.org
but now comes from registry.yarnpkg.com

let's make sure this remains the case.

**Review by ssbarbee — APPROVED** [review]

**Pull request closed without merging**